### ISSUE TITLE
Duplicate items to the end of document?

### ISSUE THREAD

**Issue by paramecie** [rgriebl/brickstore#317]
Following #311 

When you CTRL+C / CTRL+V, the item or items are copied to the END of the document.

When you duplicate some, there automatically sorted next to the original ones.

This is VERY nice, but (IMO) doesn't make it very convenient.

If I want to Duplicate those 4 items:
![image](https://user-images.githubusercontent.com/32078594/111938843-29403500-8acb-11eb-8ccb-c00b152ae4d1.png)

It gives me this mess, complicated by the fact the selected are the originals and not the new ones.

![image](https://user-images.githubusercontent.com/32078594/111938888-3d843200-8acb-11eb-8c3d-1cb010c5127e.png)

In short, as soon I've lost the selection, I've to remember then the first is to be kept as is, and the second is the copy...

![image](https://user-images.githubusercontent.com/32078594/111938994-758b7500-8acb-11eb-9e25-824f8d6d3aee.png)

Instead, just like CTRL+C/CTRL+V, simply put them at the end of document, this would show something like this, and hopefully separate the originals and the copies?

![image](https://user-images.githubusercontent.com/32078594/111939122-b5eaf300-8acb-11eb-908e-1f424fa335ef.png)

**Comment by SteveTheBrick** [comment]
I completely agree, they should go at the end of the document.

I think the new items should also be selected ready for colour,condition, etc to be changed. In fact, having the new items selected would also be great when pasting and parting out.

**Comment by chiminirc** [comment]
The way I use the software, I benefit more having the duplicate action work as it currently does where the items stay together.

**Comment by rgriebl** [comment]
I have implemented it like it is, because it is far more useful for my way of using it. Also the name "duplicate" implies that you get a clone right here and now -- not somewhere 1000s of lines away.
I do however see the benefits of switching the selection over to the duplicated lots.

What could be added though, is a "Paste special" via `Ctrl+Shift+V` that would not pop-up any messageboxes and default to "no overwrite", "no consolidation"

**Comment by paramecie** [comment]
Mmmmh could we have a global option:

**Paste, duplicate, part out**
[x] Close to the selection (if possible)
[ ] At the end of doc

But this should work the same for all paste/dups cases, and we could choose.

Just an idea, though...

**Comment by rgriebl** [comment]
I hate these options that change behavior globally, because you never know which permutation of those someone else is using: it makes it hard for
* me to debug problems (see the "visual changes" thingy interacting weirdly with "mark documents with changelogs applied as modified" we had a few days ago)

* others to help new users, because everybody's BrickStore is now behaving just a bit different here and there

**Comment by rgriebl** [comment]
I've now added a `Paste silent` command, shortcut `Ctrl+Shift+V`.

**Comment by paramecie** [comment]
Why not? But it's to be added to the contextual menu also ;-)
